Guidelines.md

```diff
@@ -1388,7 +1391,7 @@ Operation-Location: http://api.contoso.com/v1.0/operations/123
 Retry-After: 60
 ```
 
-Note: The use of the HTTP Date is inconsistent with the use of ISO 8601 Date Format used throughout this document, but is explicitly defined by the HTTP standard in [RFC 7231][rfc-7231-7-1-1-1]. Services SHOULD prefer the integer number of seconds (in decimal) format over the HTTP date format.
+Note: The use of the HTTP Date is inconsistent with the use of ISO 8601 Date Format used throughout this document, but is explicitly defined by the HTTP standard in [RFC 2616][rfc-2616-14]. Services SHOULD prefer the integer number of seconds (in decimal) format over the HTTP date format.
```

Comment:
:exclamation: I believe this was correct before (it reverts a change made in #41)